src/Map.php

```diff
@@ -3727,4 +3727,34 @@ protected function visit( iterable $entries, array &$result, int $level, ?\Closu
 			}
 		}
 	}
+
+
+	/**
+	 * Executes $callback if the Map is empty
+	 *
+	 * Examples: (Returns match or dies if doesn't exist)
+	 *  $page = $this->pages
+	 *          ->filter(fn(Page $page, $_) => $page->Id == $pageId)
+	 *          ->if(
+	 *              fn(Map &$Map) => $Map->isEmpty(),
+	 *              fn(Map &$_) => die("unknown page: " . $pageId))
```

Comment:
Please change `&$Map` to `$map` only because objects are passed by reference by default

Reply:
Resolved in 7b6ad457a2dbdf619d0e3d91ea67f009ff626a4d (also in Readme.md)

Reply:
Not yet, it's still in the method doc header

Reply:
Good spot! Must have gotten lost in the process. Fixed in c329afdda114015b900470c1ef73edc6439fc1a1